lean/deps/typed_smt2/src/galois/smt2/basic.lean

```diff
@@ -0,0 +1,185 @@
+import galois.sexpr
+
+namespace smt2
+
+/-- This represents either function or sort symbol names. -/
+def symbol := string
+
+namespace symbol
+
+/--
+Return true if this symbol is a legal name.
+
+TODO: Fix this to ensure symbol is not a reserved word, and characters are valid.
```

Comment:
This TODO makes me think we should get in the habit of filing issues.

Reply:
There's still a lot of work to be done on this.  Right now, I'd prefer to stick with comments that we can grep for.